avocado/core/job.py

```diff
@@ -90,17 +90,16 @@ def resolutions_to_tasks(resolutions, config):
     resolutions = [res for res in resolutions if
                    res.result == resolver.ReferenceResolutionResult.SUCCESS]
     no_digits = len(str(len(resolutions)))
-    sub_cmd = config.get('subcommand')
-    filter_by_tags = config.get("{}.filter_by_tags".format(sub_cmd))
+    filter_by_tags = config.get("filter_by_tags.tags")
```

Comment:
Thinking about the sections on the config file and future filters, my only suggestion here is to rename this to `filters.by_tags.tags` so we could have other filters inside the `filters` section.

Reply:
We actually had discussion on a similar topic, and have a (Trello) card about moving the filter to the (then) loader (now resolver):

https://github.com/avocado-framework/avocado/issues/3465

Which came from:

https://trello.com/c/poL0jWIi/1487-move-filter-by-tag-to-loaders

It makes sense thus to make the "filter" generic.  I'm guessing you meant that the section name would be `filter.by_tags`, right?

Reply:
@beraldoleal I've followed this up with a v2 (#3863) based on my best understanding of your suggestion here.